inductiva/tasks/task.py

```diff
@@ -174,13 +175,73 @@ def wait(self, polling_period: int = 5) -> models.TaskStatusCode:
 
             time.sleep(polling_period)
 
-    def kill(self) -> None:
-        """Kill the task.
+    def kill(
+            self,
+            wait_timeout: Optional[Union[float,
+                                         int]] = None) -> Union[bool, None]:
```

Comment:
Isn't there a way to prevent these weird line breaks?

Reply:
Yapf won't allow me to do it.